src/redux/slices/videogamesSlice.js

```diff
@@ -68,22 +73,16 @@ const videogamesSlice = createSlice({
         price_per_day: 35,
       },
     ],
-    details: {
-      id: 1,
-      name: 'Super Mario Bros',
-      photo: 'https://cdn.mobygames.com/covers/4039218-super-mario-bros-nes-front-cover.jpg',
-      description: 'Super Mario Bros is a platform game developed and published by Nintendo. The successor to the 1983 arcade game Mario Bros. and the first game in the Super Mario series, it was first released in 1985 for the Famicom in Japan.',
-      pricePerDay: 5,
-    },
+    details: {},
     message: null,
     error: null,
   },
   extraReducers: {
-    [getDetails.fulfilled]: (state, action) => {
-      state.details = action.payload;
+    [getDetails.fulfilled]: (state, { payload }) => {
+      state.details = payload;
     },
     [getDetails.rejected]: (state, action) => {
-      state.details = action.payload;
+      state.error = action.payload;
     },
     [getDetails.pending]: (state, action) => {
```

Comment:
Do we need this extraReducer? I'm thinking we don't because we're rendering "loading" when there are no details. If we choose to remove it, we would need to initialize details to `null` or `undefined` and it should work.